src/modules/public/configs/sagas/configsFetchSaga.test.ts

```diff
@@ -67,7 +63,14 @@ describe('Saga: configsFetchSaga', () => {
     it('should trigger an error on configs fetch', async () => {
         const expectedActions = [
             configsFetch(),
-            configsError(alertDataPayload),
+            sendError({
+                error,
+                processingType: 'alert',
+                extraOptions: {
+                    type: CONFIGS_FETCH,
```

Comment:
`type` property should be optional, you don't have to pass it on each `sendError` call.
`type` property should be passed only when it's handled inside `sendError`

Reply:
Done